Dockerfile

```diff
@@ -16,20 +17,15 @@ COPY setup-r-envt.R setup-r-envt.R
 
 RUN Rscript setup-r-envt.R
 
-RUN conda config --add channels conda-forge
-
-RUN conda config --set channel_priority strict
-
 COPY environment.yml environment.yml
-
-RUN conda update -n base -c defaults conda
-
-RUN conda env update -n root -f environment.yml
+RUN conda config --add channels conda-forge \
+    && conda config --set channel_priority strict \
+    && conda update --all --yes \
+    && conda env update -n base -f environment.yml \
+    && conda clean --all --yes --force-pkgs-dirs
```

Comment:
`--all --yes` is = `tipsy` and `--force-pkgs-dirs` get us some extra free space.